### DISCUSSION TITLE
code action for return type generation

### DISCUSSION THREAD

**Discussion by scarf005** [microsoft/pylance-release#3486] [Enhancement]
return type of some function can be automatically inferred. for example:

```py
def add(a: str, b: str):
    return a + b
```
![image](https://user-images.githubusercontent.com/54838975/196021266-9a96e8a5-f900-469a-9213-2753430675d2.png)

It would be useful to have a code action that adds inferred return type to function definition.

**Comment by bschnurr** [comment]
for now there is an inlayhint setting.. where you can double click on the hint to insert the annotation.
![image](https://user-images.githubusercontent.com/1946977/196235062-936de764-f5c7-4375-a962-2f58e7fdbd23.png)

**Comment by bschnurr** [comment]
We were looking to add a code action to insert as well.
